### PR TITLE
Add transaction submission route to T&T Client Server

### PR DESCRIPTION
**Pull request by delventhalz**
To test you must:
1. Create a user by submitting a JSON `POST` request to `localhost:3000/api/users`:
```
{
  "username": <string>,
  "password": <string>,
  "email": <string, email>,
  "publicKey": <string, valid public key>
}
```
2. Save the `authorization` string that is returned.
3. Create a transaction signed by the with the same key
4. Encode in a `TransactionList` and submit as the body in a `POST` request to `localhost:3000/api/transactions`. Set the headers to:
```
"Authorization": <string, the authorization key>
"Content-Type": "application/octet-stream"
```

### CODE REVIEW

**Comment by delventhalz** [comment]
@peterschwarz You submit it in the POST to create the user. The server is agnostic about how you generate it, but once you've created a user account with a public key, it will expect that key to match for submitted transactions.

The next PR will be login/signup which will generate public keys client side used the SDK.

**Comment by peterschwarz** [comment]
The batching pubkey is different from the client pubkey, but needs to be included in the transaction header.  Is it returned to the user on the initial authorization task?

**Comment by delventhalz** [comment]
@peterschwarz Ah. That is a good question. My thought was it would just be a configuration variable, that whoever was deploying the client would set up. I'm not sure sending it back on authorization makes much sense since it will be the same for all users. Perhaps an API route which will send back the publicKey? `GET /api/config` or some such?

File: families/track_and_trade/server/blockchain/deltas.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+const getProtoName = address => {
+  const typePrefix = address.slice(6, 8)
+  if (typePrefix === 'ea') {
+    if (address.slice(-4) === '0000') return 'Property'
+    else return 'PropertyPage'
+  }
+
+  const names = {
+    ae: 'Agent',
+    aa: 'Property',
```
Comment (nick-drozd) [inline families/track_and_trade/server/blockchain/deltas.js]: 'Proposal'

**Pull request merged**